### PR TITLE
Use stalebot on this repo

### PR DESCRIPTION
**Pull request by kjdelisle**
### Description

Stalebot will clean up stale issues after a specified time.
See the .github/stale.yml file for details.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
```
Comment (b-admike) [inline .github/stale.yml]: How about `major` or `p1` here (if they exist)?

Comment (loay) [inline .github/stale.yml]: I think Critical plays the same role as Major

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - security
+  - critical
+# Label to use when marking an issue as stale
+staleLabel: wontfix
```
Comment (virkt25) [inline .github/stale.yml]: I think we should mark them as `stale` and not `wontfix`

File: .github/stale.yml (hunk 1 above)
Comment (b-admike) [inline .github/stale.yml]: We have two different labels though -_-

Comment (kjdelisle) [inline .github/stale.yml]: Technically, they're two different categories of severity, critical being the highest and major the 2nd highest.

**Comment by kjdelisle** [comment]
Tests already passed before and it's not under test, so merging. Thanks for your reviews.

**Pull request merged**